go-packages/meep-logger/logger.go

```diff
@@ -56,6 +56,10 @@ func getLogCaller() string {
 	return location
 }
 
+func GetComponentName() string {
+	return componentName
+}
+
```

Comment:
Remove this. Instead, inject module name in new HTTP logger instance.

Reply:
done